.github/workflows/lint.yaml

```diff
@@ -0,0 +1,29 @@
+name: 🕵 Lint
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Shouldn't be `secrets.TEST_GITHUB_PAT`?

Reply:
This job does not require custom GitHub token, as it does not make use of GitStats.